code/controllers/evacuation/evacuation_pods.dm

```diff
@@ -48,26 +44,15 @@
 				pod.launch(src)
 
 		priority_announcement.Announce(replacetext(replacetext(GLOB.using_map.emergency_shuttle_leaving_dock, "%dock_name%", "[GLOB.using_map.dock_name]"),  "%ETA%", "[round(get_eta()/60,1)] minute\s"))
-	else
```

Comment:
what exactly does this removed block do

Reply:
It was related to the state that is applied upon bluespace jump. It duplicates certain mobs as a hallucination while in bluespace, and movement becomes very uncoordinated.